src/Generator/Generators/ExtensionMethods.cs

```diff
@@ -1,10 +1,17 @@
-﻿using CppSharp.AST;
+﻿using System.Linq;
+using System.Collections.Generic;
+using CppSharp.AST;
+using CppSharp.AST.Extensions;
 using Interop = System.Runtime.InteropServices;
 
 namespace CppSharp.Generators
 {
     public static class ExtensionMethods
     {
+        public static List<PrimitiveType> AllowedToHaveDefaultPtrVals = new List<PrimitiveType> {PrimitiveType.Bool, PrimitiveType.Double, PrimitiveType.Float,
```

Comment:
Now that you have a dedicated function, you could make this private or better yet, make it local for the function.